.changeset/shaggy-stingrays-look.md

```diff
@@ -0,0 +1,6 @@
+---
+'@aws-amplify/integration-tests': minor
+'@aws-amplify/cli-core': minor
```

Comment:
doesn't the template in create-amplify also need an update?

Reply:
create-amplify is using `package_manager_controller_base.ts` for initializing tsconfig so there are no changes there for this